src/components/Story/Story.js

```diff
@@ -46,18 +57,20 @@ class Story extends React.Component {
   };
 
   handleClick = (key) => {
+    const { post } = this.props;
+
     switch (key) {
       case 'follow':
-        this.props.onFollowClick(this.props.post);
+        this.props.onFollowClick(post);
         return;
```

Comment:
Is there a reason to use `return` here vs `break` as in all other ones?

Reply:
No good reason, it was this way before this PR. It's not big change so I will change this here.